services/src/messaging/src/main/java/org/openkilda/messaging/command/flow/FlowReadRequest.java

```diff
@@ -22,18 +22,22 @@
 import lombok.Value;
 
 /**
- * Represents get bidirectional flow northbound request.
+ * Represents flow read northbound request.
  */
 @Value
-public class BidirectionalFlowRequest extends CommandData {
+public class FlowReadRequest extends CommandData {
 
     private static final long serialVersionUID = 1L;
 
     @JsonProperty("flowid")
     protected String flowId;
 
+    public FlowReadRequest() {
```

Comment:
This is not obvious what empty flowId means and doesn't correlate to the request name. Why not create a separate request to fetch all flows.

Reply:
Reasonable